pkg/kubelet/eviction/helpers.go

```diff
@@ -137,9 +134,35 @@ func ParseThresholdConfig(allocatableConfig []string, evictionHard, evictionSoft
 			}
 		}
 	}
+	enforceAllocatable := false
+	for _, key := range allocatableConfig {
+		if key == kubetypes.NodeAllocatableEnforcementKey {
+			enforceAllocatable = true
+		}
+	}
+	if enforceAllocatable {
+		results = addAllocatableThresholds(results)
```

Comment:
nit: could move this inside the above for loop and break afterward, removing the need for `enforceAllocatable`

Reply:
done